tests/system_tests/test_exceptions.py

```diff
@@ -266,9 +266,10 @@ def run_test_breaking_into_raised_and_unhandled_exceptions(
             Awaitable.wait_all(stopped2, continued)
 
             # Second hit on uncaught exception
-            self.assertEqual(stopped2.event.body['text'], 'ArithmeticError')
-            self.assertIn("ArithmeticError('Hello'",
-                          stopped2.event.body['description'])
+            # TODO: Fix in #2294
```

Comment:
>2294 [](start = 28, length = 4)

is the bug number correct? if so which repo?

Reply:
lol in VSC